migrations.py

```diff
@@ -37,40 +39,51 @@ async def m002_major_overhaul(db: Database):
     # Initial settings table
     await db.execute(
         """
-        CREATE TABLE discordbot.settings (
+        CREATE TABLE IF NOT EXISTS discordbot.settings (
             admin TEXT PRIMARY KEY,
-                CONSTRAINT admin_account_id 
-                FOREIGN KEY(admin)
-                REFERENCES accounts(id)
-                ON DELETE cascade,
-            bot_token TEXT NOT NULL UNIQUE
+            bot_token TEXT NOT NULL UNIQUE,
+            CONSTRAINT admin_account_id 
+            FOREIGN KEY(admin)
+            REFERENCES accounts(id)
+            ON DELETE cascade
         );
     """
     )
     # Migrate old data
     if db.type == SQLITE:
+        um_db = Database("ext_usermanager")
+        rows = await db.fetchall("SELECT * FROM users")
```

Comment:
- this is not allowed anymore
- only APIs should be used

Reply:
I thought of using the api aswell but I dont think it would work because I dont need to create new data but instead migrate the exact old data